config/packages/prod/messenger.yml

```
@@ -0,0 +1,22 @@
+framework:
+    messenger:
+        transports:
+            # Producers
+            business_event:
+                dsn: doctrine://default
+                options:
+                    table_name: messenger_messages
+                    queue_name: default
+                    redeliver_timeout: 3600
+                    auto_setup: false
+                serializer: pim_enrich.messenger.serializer.business_event
+
+            # Consumers
+            webhook:
+                dsn: doctrine://default
+                options:
+                    table_name: messenger_messages
+                    queue_name: default
+                    redeliver_timeout: 3600
+                    auto_setup: false
+                serializer: pim_enrich.messenger.serializer.business_event
```

The Doctrine Transport is a standard queue (not like the subscription model of pubsub) so we only need to have the `webhook` as input source (via the routing) and output.
```suggestion
            webhook:
                dsn: doctrine://default
                options:
                    table_name: messenger_messages
                    queue_name: default
                    redeliver_timeout: 3600
                    auto_setup: false
                serializer: pim_enrich.messenger.serializer.business_event

        routing:
            'Akeneo\Platform\Component\EventQueue\Event': webhook
            'Akeneo\Platform\Component\EventQueue\BulkEvent': webhook
```

---

updated ! thanks @pchasle